### DISCUSSION TITLE
Changing Active Color of a Menu component

### DISCUSSION THREAD

**Discussion by FreakQnZ** [saadeghi/daisyui#2488] [Q&A] [answered]
When I click on any item, on mouseclick (active) it gives a dark bg color, how do I change that

![image](https://github.com/saadeghi/daisyui/assets/140912142/14f8d0d1-8e07-43c2-a636-ee1e0034927f)

this is the code 

![image](https://github.com/saadeghi/daisyui/assets/140912142/a556d0af-c4c7-44cf-a155-c240acef6a4f)

**Comment by saadeghi** [comment, marked as answer]
Here's a way: https://play.tailwindcss.com/MqmtEOwInW?file=css  
Or: https://play.tailwindcss.com/zxPKfvjok7

**Comment by Vishu-hp** [comment]
You can override active class styles like this :
![helper](https://github.com/user-attachments/assets/0a4eac24-ad5d-42f0-9169-88d303b1b40e)
